plugins/vswitch/wayfire/plugins/vswitch.hpp

```diff
@@ -191,8 +192,8 @@ class workspace_switch_t
                 wf::TRANSFORMER_2D, vswitch_view_transformer_name);
             wf::scene::set_node_enabled(view->get_transformed_node(), false);
 
-            // Render as an overlay, but make sure it is translated and clipped properly to the local output
-            overlay_view_node = std::make_shared<scene::output_node_t>(output);
+            // Render as an overlay, but make sure it is translated to the local output
+            overlay_view_node = std::make_shared<scene::translation_node_t>(output);
```

Comment:
This may look correct but is slightly misleading. The translation node has a boolean parameter (which in this particular case can be either true or false, it won't make a difference), and since output is a pointer, it is auto-converted to a bool.

You need to set the node's offset manually to origin(o->get_layout_geometry())

Reply:
> This may look correct but is slightly misleading. The translation node has a boolean parameter (which in this particular case can be either true or false, it won't make a difference), and since output is a pointer, it is auto-converted to a bool.

Oops! Fixed.

> You need to set the node's offset manually to origin(o->get_layout_geometry())

Attempted.